### PR TITLE
Reuse Openai client

### LINKED ISSUE #42: OpenAI client enhancement
## Short description of current behavior
The goal of this issue is to add code to create the client once and reuse during multiple chat completion queries

## Steps 🕵️‍♂️ 🕵️‍♀️
- You can familiarize yourself with [Minds](https://mindsdb.com/).

- Create Pull Request to this repo.

## 📢 Additional rewards 🏆
This issue brings 1️⃣ 0️⃣  points for SWAG and entry into the draw for a 👇 :

3️⃣ ❌ 💻 [Razer Blade 16 Gaming Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0510TEN3-R3U1)
👕 🐻 [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)
ℹ️ For more info check out https://mindsdb.com/hacktoberfest 👈

### PR DESCRIPTION
**Pull request by Better-Boy**
Reuse openai client object for chat completion

Fixes #42

### CODE REVIEW

File: minds/utils.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from urllib.parse import urlparse, urlunparse
+
+def create_base_url_openai(base_url: str) -> str:
```
Comment (ea-rus) [inline minds/utils.py:3]: maybe rename to `get_openai_base_url` or `gen_openai_base_url`

**Review by ea-rus — APPROVED** [review]

File: minds/utils.py (hunk 1 above)
Comment (Better-Boy) [reply minds/utils.py]: Done

**Comment by Better-Boy** [comment]
@ea-rus can you merge the PR as it's approved?

File: minds/minds.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
 from typing import List, Union, Iterable
-from urllib.parse import urlparse, urlunparse
-
+import utils
```
Comment (ea-rus) [inline minds/minds.py:2]: `utils` is imported below from mind

**Pull request merged**